### DISCUSSION TITLE
@loopback/logging

### DISCUSSION THREAD

**Discussion by cuttingd** [loopbackio/loopback-next#9265] [Q&A] [answered]
Is there a timeframe when it's expected that @loopback/logging will no longer be in "Experimental" state?

**Comment by achrinza** [comment, marked as answer]
Hi @cuttingd, there's no timeline on bringing `@loopback/logging` to v1 / non-experimental.

This is partly due to the lack of discussion surrounding the package's design. Hence, there is a higher chance that we may expand the package's capabilities beyond what was initially conceived.

However, it should be fine to use it in production as:

1. There hasn't been much historical breaking changes for this package
2. It's a thin integration between LoopBack 4 and Winston
3. Rarely has entire LoopBack 4 packages been "deprecated". I'm only aware of 1 package that was removed from the GitHub repository, and that was pre-GA LoopBack 4.

Hence we do not foresee any major changes to this package for the foreseeable future.

As long as the NPM-default caret SemVer range is used (e.g. "^0.1.0"), no breaking changes would be installed.

As a general principle LoopBack 4 is maintained with a less rapid development cycle. That is, LoopBack 4 maintainers do not make sweeping breaking changes nor do we remove existing functionality without good reason.

Hope this helps!

**Reply by cuttingd** [reply]
Thank you
